@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardHostsPatches.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiCard>
+    <UiCardTitle class="patches-title">
+      {{ $t("patches") }}
+      <template v-if="areAllLoaded" #right>
+        {{ $t("missing", { n: count }) }}
+      </template>
+    </UiCardTitle>
+    <div class="table-container">
+      <HostPatches :hosts-loaded-status="loadedStatus" :patches="patches" />
+    </div>
+  </UiCard>
+</template>
+
+<script lang="ts" setup>
+import HostPatches from "@/components/HostPatchesTable.vue";
+import UiCard from "@/components/ui/UiCard.vue";
+import UiCardTitle from "@/components/ui/UiCardTitle.vue";
+import { useHostPatches } from "@/composables/host-patches.composable";
+import { useHostStore } from "@/stores/host.store";
+import { computed } from "vue";
+
+const { records: hosts } = useHostStore().subscribe();
+
+const { loadedStatus, count, patches } = useHostPatches(() =>
+  hosts.value.map((host) => host.$ref)
+);
+
+const areAllLoaded = computed(() => {
+  return [...loadedStatus.value.values()].every((isReady) => isReady);
+});
```

Comment:
Sorry I missed this comment.

It is indeed possible to `expose` the _computed_ from `HostPatchesTable.vue` but in general I don't like to create a dependency between components.

However, since in Vue exposed variable ares clearly defined I think it's acceptable.

I'll made the change.